### PR TITLE
Remove `io.ballerina.runtime.internal.util.exceptions.BLangRuntimeException` usage

### PR DESCRIPTION
**Pull request by Nadeeshan96**
## Purpose

$title

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Dependencies.toml [hunk 1]
```diff
@@ -17,7 +18,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "observe"
-version = "1.0.7"
+version = "1.0.8"
```
Comment (gabilang) [inline ballerina/Dependencies.toml:21]: Do we need to change the version here?

File: ballerina/Dependencies.toml [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 [ballerina]
 dependencies-toml-version = "2"
+distribution-version = "2201.6.0-20230519-135600-d887142d"
```
Comment (gabilang) [inline ballerina/Dependencies.toml:8]: Should this distribution-version be needed here?

File: ballerina/Ballerina.toml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 org = "ballerina"
 name = "observe"
-version = "1.0.7"
+version = "1.0.8"
 distribution = "slbeta3"
 export = ["observe", "observe.mockextension" ]
 
 [[platform.java11.dependency]]
-path = "../native/build/libs/observe-native-1.0.7.jar"
+path = "../native/build/libs/observe-native-1.0.8-SNAPSHOT.jar"
```
Comment (gabilang) [inline ballerina/Ballerina.toml:9]: Shouldn't we need to add a time-stamp version here?

Comment (Nadeeshan96) [reply ballerina/Ballerina.toml:9]: These changes are auto-generated when building the observe module according to the version information in `gradle.properties` file. Commit: 7e40001e7ac0595230ca495a8085211f2319159f. I think it is okay to keep them as this is usually done in stdlib modules also.

File: ballerina/Dependencies.toml (hunk 2 above)
Comment (Nadeeshan96) [reply ballerina/Dependencies.toml:8]: These changes are auto-generated when building the observe module according to the version information in `gradle.properties` file. Commit: 7e40001e7ac0595230ca495a8085211f2319159f. I think it is okay to keep them as this is usually done in stdlib modules also.

File: ballerina/Dependencies.toml (hunk 1 above)
Comment (Nadeeshan96) [reply ballerina/Dependencies.toml:21]: These changes are auto-generated when building the observe module according to the version information in `gradle.properties` file. Commit: 7e40001e7ac0595230ca495a8085211f2319159f. I think it is okay to keep them as this is usually done in stdlib modules also.

**Review by warunalakshitha — APPROVED** [review]
LGTM

**Pull request merged**